cmd/bosun/database/database.go

```diff
@@ -19,7 +19,7 @@ import (
 	"github.com/captncraig/easyauth/providers/token/redisStore"
 )
 
-var SchemaVersion = int64(1)
+var SchemaVersion = int64(2)
```

Comment:
I think perhaps I could just wrap this in an init() and do the len of `tasks` for the value of `SchemaVersion`, but I have some feeling that perhaps there was a reason I didn't do this in the first place - also I don't feel that motivated to mess with it right now.